### PR TITLE
Feature/global ens drivers naefs stats

### PR DESCRIPTION
**Pull request by StevenSimon-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>
All updated global_ens atmos gefs, cmce, ecme, and naefs prep, stats, and plots processing has been tested interactively and confirmed. The primary focus is the expansion of the naefs g2o acc verification statistic and plotting of the expanded naefs acc verification statistic to include raw gefs and cmce fhrmean series for 850 hPa temp and 850 hPa and 250 hPa u-wind and v-wind. The secondary focus is the removal of the "exclhost" tag from the memory settings of stats and plots drivers with fewer than 128 cpu's. The "exclhost" tag was added to the memory settings of profile4 and naefs precip drivers following the increase of their cpu nodes to 128 nodes.   
- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Run all atmos prep, stats, and plots drivers, except for the headline plots driver. The headline plots driver is to be run only once per year, but will be retained in EVS v1 as something that does not run in ops daily. Check that F000 stats files are being generated for all global_ens g2g and g2o fields. 

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**
Yes

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**
Yes

- [ ] Please complete this pull request review by **[Fill in date]**.</br>
ASAP

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

Overview: This PR addresses the expansion of the naefs g2o acc verification metric and derived plots to include fhr mean series for raw gefs and raw cmce. Thus, these g2o plots now have the naefs, raw gefs, and raw cmce fhrmean series on the same plots of these metrics. This PR also removes unnecessary "exclhost" tags from stats and plots drivers with memory settings of nodes fewer than 128 cpu's. The amount of nodes was increased to 128 cpu's for the profile4 and naefs precip plots drivers and the "exclhost" tag added to their memory settings thereafter. This was done to address the memory related errors associated with these particular plots drivers. Also, F000 stats generation capabilities were added to the global_ens atmos (gefs, cmce, ecme, and naefs) fields which do not currently have them.

### CODE REVIEW

File: parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf
Comment (malloryprow) [inline parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: I caught this in some of the other config files but 

LOG_LEVEL = DEBUG
LOG_GEN_ENS_PROD_VERBOSITY = 2

are not needed as this stuff gets defined in machine.conf!

Comment (StevenSimon-NOAA) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: Is it necessary to delete these two lines in just this config file or for the others Binbin edited?

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: The others ones too.

Comment (StevenSimon-NOAA) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: Just the four Binbin edited?

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: Yes it looks like those type of logging settings are in the files.

Comment (StevenSimon-NOAA) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: After making this modification, none of my waves plots are being generated from the plots driver script. Could you please review the tracked changes I made in these four config files to make sure that these modifications are not the issue?

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: This file is completely unrelated to the global_ens wave plots; this is related to the global_ens atmos stats.

Comment (PerryShafran-NOAA) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: I wonder if LOG_GEN_ENS_PROD_VERBOSITY should remain.  That specific variable is not in the machine.conf file, but more a general verbosity statement.  Try putting the verbosity statements back in and see what happens.

LOG_LEVEL is definitely in machine.conf and doesn't need to be added.

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: I wouldn't see how a METplus config file change for global_ens atmos stats would impact global_ens wave plots in anyway. Additionally, we set the logging values in machine.conf so they would be standardized across EVS. While these are the same values, we shouldn't be setting these in the METplus config files.

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-LOG_GEN_ENS_PROD_VERBOSITY

Comment (malloryprow) [reply parm/metplus_config/global_ens/atmos_grid2obs/stats/GenEnsProd_fcstCMCE_obsPREPBUFR_UPPER_climoERA5.conf:1]: Issue is unrelated to the changes in this file. "import os" needs to be added in ush/global_ens/ush_gens_plot_py/settings.py and ush/global_ens/ush_gens_plot_py/settings.py.

File: ush/global_ens/settings.py
Comment (malloryprow) [inline ush/global_ens/settings.py:1]: Add "import os" at the top of this file.

File: ush/global_ens/ush_gens_plot_py/settings.py
Comment (malloryprow) [inline ush/global_ens/ush_gens_plot_py/settings.py:1]: Add "import os" at the top of this file.

Comment (StevenSimon-NOAA) [reply ush/global_ens/ush_gens_plot_py/settings.py:1]: Done and applied to both global_ens settings.py ush files.

**Comment by ShelleyMelchior-NOAA** [comment]
I thought I would be sneaky this morning and get the global_ens prep runs in before cactus was configured for prod. But darnit! Will have to wait till later.
`$ qsub /lfs/h2/emc/vpppg/noscrub/shelley.melchior/githubwkspc/forks/SSi/EVS/dev/drivers/scripts/global_ens/prep/jevs_global_ens_atmos_prep.sh`
`qsub: Queue is not enabled`
`Job submit error: 173.`

**Comment by ShelleyMelchior-NOAA** [comment]
The prep processing is completed.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_prep.o77973094`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_atmos_prep.o77973093`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_prep.o77973095`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/atmos.20230821`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/wave.20230822`

NAEFS completed pretty rapidly, ~3 minutes.
wave completed in ~15 minutes.
GEFS completed in ~4 hrs and 12 minutes.

**Comment by ShelleyMelchior-NOAA** [comment]
@StevenSimon-NOAA , do I need to use my newly generated prep files as COMIN for the stats processing to properly test the f000 situation? Or can I use the more complete set of prep data from the EVS parallel?

**Comment by StevenSimon-NOAA** [comment]
> @StevenSimon-NOAA , do I need to use my newly generated prep files as COMIN for the stats processing to properly test the f000 situation? Or can I use the more complete set of prep data from the EVS parallel?

It's my understanding that the F000 prep files for gefs, naefs, and cmce are currently not available in the EVS Parallel's COMOUT. Only the ecme has F000 prep files in the EVS Parallel's COMOUT. With that in mind, we may need to use the newly generated prep files as COMIN for the stats profile.

**Comment by ShelleyMelchior-NOAA** [comment]
The prep processing has concluded. I updated my original comment in this PR conversation. Since this is a big PR, while Steve reviews the prep results, I will commence w/ the stats processing and I will use the prep dataset that I just generated so that the f000 statistics can be generated.

**Comment by ShelleyMelchior-NOAA** [comment]
I started wave stats. It completed in ~8 mins, keeping in mind that the prep data is limited.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/wave.20230822`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_stats.o78043697`

I will submit the other stats processing for atmos.

**Comment by StevenSimon-NOAA** [comment]
> I started wave stats. It completed in ~8 mins, keeping in mind that the prep data is limited. COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/wave.20230822` logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_stats.o78043697`
> 
> I will submit the other stats processing for atmos.

It looks like Mallory recently had a PR merged that may affect the global_ens wave stats: Remove FBIAS==1 and other thresholds from wave components. Please proceed forward or rerun if necessary.

Update: I just merged this PR into the relevant feature branch.

**Comment by StevenSimon-NOAA** [comment]
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/wave.20230822

All prep files, including the new F000 ones, appear to be all accounted for. Note that we are one prep file short for global_ens gefs atmos prep, but that is because your global_ens archive does not have two consecutive days of gefs atmos prep data. So that result is expected.

**Comment by ShelleyMelchior-NOAA** [comment]
Got it re: prep. Should I run prep again today so that the prep data has 2 consecutive days?

I pulled in the updates and I will re-ran the wave stats.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_stats.o78050473`

**Comment by StevenSimon-NOAA** [comment]
> Got it re: prep. Should I run prep again today so that the prep data has 2 consecutive days?
> 
> I pulled in the updates and I will re-ran the wave stats. logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_stats.o78050473`

If we have enough time to do so, I say let's do it.

**Comment by ShelleyMelchior-NOAA** [comment]
global_ens prep processing is underway again to get 2 days of prep data files.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_atmos_prep.o78091981`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_prep.o78091315`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_prep.o78091058`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/atmos.20230822`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/wave.20230823`

**Comment by ShelleyMelchior-NOAA** [comment]
2 days of naefs prep data files are available, so I have run naefs stats processing. Processing has completed.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_precip_stats.o78093528`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_grid2grid_stats.o78093526`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_grid2obs_stats.o78093527`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/naefs.20230822`

**Comment by StevenSimon-NOAA** [comment]
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/atmos.20230822

NAEFS prep files, including the new F000 ones, are all accounted for in the COMOUT directory.

**Comment by ShelleyMelchior-NOAA** [comment]
2 days of wave prep data files are available, so I have run wave stats processing. 
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_wave_grid2obs_stats.o78096315`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/gefs.20230823`

**Comment by StevenSimon-NOAA** [comment]
> /lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/wave.20230823

All GEFS wave prep files, including the F000 ones, are all accounted for in the COMOUT directory. Similarly, all GEFS wave stats files, including the F000 ones, are all accounted for in the COMOUT directory.

**Comment by StevenSimon-NOAA** [comment]
> 2 days of naefs prep data files are available, so I have run naefs stats processing. Processing has completed. logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_precip_stats.o78093528` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_grid2grid_stats.o78093526` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_atmos_grid2obs_stats.o78093527` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/naefs.20230822`

All naefs g2g, g2o, and precip stats files, including those for the F000 stats, are fully accounted for in the COMOUT directory. No memory oversubscription errors in their associated log files.

**Comment by ShelleyMelchior-NOAA** [comment]
The 2nd run of gefs atmos prep is nearly complete. Heading to bed, but this has been running for 4.5 hours, so it should be done soon! This will get us 2 days worth of prep to then run the gefs atmos stats.
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_atmos_prep.o78091981`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/`

**Comment by ShelleyMelchior-NOAA** [comment]
FYI -- lost power this morning after some nasty overnight storms. I am not sure when I will be restored. Glad I worked a bit into the evening last night.

UPDATE: power is restored. I will try to kick off the global_ens atmos stats processing today.

**Comment by StevenSimon-NOAA** [comment]
> The 2nd run of gefs atmos prep is nearly complete. Heading to bed, but this has been running for 4.5 hours, so it should be done soon! This will get us 2 days worth of prep to then run the gefs atmos stats. logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_atmos_prep.o78091981` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/prep/global_ens/`

All global_ens atmos prep files, including the new F000 ones, are fully accounted for in the COMOUT directory. Ditto for the naefs prep files. jevs_global_ens_atmos_prep.o78091981 indicates the memory oversubscription issue.

**Comment by ShelleyMelchior-NOAA** [comment]
Stats for cmce have completed. Please review.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_cmce_atmos_precip_stats.o78176834`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_cmce_atmos_snowfall_stats.o78176835`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_cmce_atmos_grid2grid_stats.o78176832`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_cmce_atmos_grid2obs_stats.o78176833`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/cmce.20230822`

**Comment by ShelleyMelchior-NOAA** [comment]
Stats for ecme have completed. Please review.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_precip_stats.o78182180`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_snowfall_stats.o78182117`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2grid_stats.o78182226`
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2obs_stats.o78182247`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/ecme.20230822`

**Comment by StevenSimon-NOAA** [comment]
> Stats for ecme have completed. Please review. logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_precip_stats.o78182180` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_snowfall_stats.o78182117` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2grid_stats.o78182226` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2obs_stats.o78182247` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/ecme.20230822`

All cmce g2g, g2o, precip, and snowfall stats files, including those for the F000 stats, are fully accounted for in the COMOUT directory. No memory oversubscription errors in their associated log files.

**Comment by StevenSimon-NOAA** [comment]
> Stats for ecme have completed. Please review. logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_precip_stats.o78182180` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_snowfall_stats.o78182117` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2grid_stats.o78182226` `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_ecme_atmos_grid2obs_stats.o78182247` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/ecme.20230822`

All ecme g2g, g2o, precip, and snowfall stats files, including those for the F000 stats, are fully accounted for in the COMOUT directory. No memory oversubscription errors in their associated log files.

**Comment by ShelleyMelchior-NOAA** [comment]
Stats for gefs have completed (except gefs_atmos_cnv is still running). Please review.
logfiles:
`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_gefs_atmos_*_stats.o*`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822`
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/gefs.20230822`

**Comment by StevenSimon-NOAA** [comment]
> Stats for gefs have completed (except gefs_atmos_cnv is still running). Please review. logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_gefs_atmos_*_stats.o*` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/atmos.20230822` COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/global_ens/gefs.20230822`

All gefs g2g, g2o, precip, sea ice, snowfall, sst, and cnv stats files, including those for the F000 stats, are fully accounted for in the COMOUT directory. There are some empty cnv stats files at F030, F036, and F042, but that may be a function of the small sample size of prep date in the COMOUT archive. I do not have this issue with my larger prep files archive. No memory oversubscription errors in their associated log files.

**Comment by ShelleyMelchior-NOAA** [comment]
Re-running global_ens_atmos_prep to test the oversubscription correction.
logfile:`/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_atmos_prep.o78201755`

**Comment by ShelleyMelchior-NOAA** [comment]
While global_ens_atmos_prep retest churns away, I got started on naefs plots.
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/global_ens/atmos.20230822`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_*_plots.o*`

**Comment by StevenSimon-NOAA** [comment]
> While global_ens_atmos_prep retest churns away, I got started on naefs plots. COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/global_ens/atmos.20230822` logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_naefs_*_plots.o*`

The F000 data is not being plotted, even though the x-axis allows space for it. FYI, all naefs plots are fhrmean plots.
I do not think the plots are going to automatically update.
The good news is that all three model series are being plotted for acc.

**Comment by StevenSimon-NOAA** [comment]
I adjusted the memory settings downward on the naefs precip plot drivers. The exclhost 128GB setting was unnecessary for them to run successfully to completion.

**Comment by ShelleyMelchior-NOAA** [comment]
I ran the profile global_ens plots. I used VDATE=20230823 and used my local (and limited) stats files. Please review:
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_profile*.o7841*`
COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/global_ens/atmos.20230823`

I also ran the precip plots, specifically the spatial plot, to see if it required the met and metplus modules. It indeed does. So I will not modify the modulefiles for the global_ens component.

**Comment by StevenSimon-NOAA** [comment]
> I ran the profile global_ens plots. I used VDATE=20230823 and used my local (and limited) stats files. Please review: logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_global_ens_profile*.o7841*` COMOUT=`/lfs/h2/emc/ptmp/shelley.melchior/evs/v1.0/plots/global_ens/atmos.20230823`
> 
> I also ran the precip plots, specifically the spatial plot, to see if it required the met and metplus modules. It indeed does. So I will not modify the modulefiles for the global_ens component.

Time series plots are not being generated for profile1 and profile2 plots drivers, only fhrmean plots. I do not know if that is expected, given the limited amount of stats data. As an aside, my side interactive runs of the same profile1 and profile2 plots drivers generates time series plots in addition to the fhrmean plots. But if I recall correctly, the first set of time series plots are not generated until f120 stats data are available. The limited stats archive does not have those yet.  The profile3 and profile4 stats drivers generate vertprof plots out to 36 hrs. That is expected. However, the f000 vertprof plots were not generated. This may have to be taken care of on the global_ens plots PR Mallory has been working on.

**Comment by StevenSimon-NOAA** [comment]
On a positive note, it looks the memory oversubscription issue has been resolved in the global_ens atmos prep driver and the profile plots drivers. 

I am going to be away from my computer for the rest of the day, but will check in on any updates tomorrow morning. The F000 stats are now being generated. The next step is getting that F000 stats data onto plots. 

The dev-prod switch schedule makes it challenging to maintain a continuous prep, stats, and plots archive. The outages throw another wrench into these efforts. One has to copy over large quantities of prep and stats data from machine to machine to ensure this continuity. As more data accumulates, this become more and more of a time consuming and tedious exercise, especially if you cannot simply transfer over data from prod machine to dev machine directly. Unfortunately, without a continuous archive of successful prep and stats runs, there is no guarantee that all expected up-to-date plots can and will be generated every day. Not being able to use the EVS Parallel's extensive prep and stats archive in development activities, such as those with this and similar PRs, adds to those challenges.

**Comment by ShelleyMelchior-NOAA** [comment]
@StevenSimon-NOAA , I can use the emc.vpppg stats for the plots. I just hadn't been since those stats do not have f000 calculated.

As for your personal archive, where there are holes, you can try to fill them w/ emc.vpppg files as a stop gap. Otherwise, like @ShannonShields-NOAA  does, you need to stage the prep data in /dfs/write on cactus ahead of the switch. Then once the switch is complete, you can grab the prep data from /dfs/read on dogwood to populate your archive. One problem, though, is that the /dfs disk is limited. If you cannot get all of your prep data staged in /dfs/write on cactus (you can always regenerate stats), ask me and Alicia and we can try to stage the remainder in our /dfs/write spaces.

**Comment by StevenSimon-NOAA** [comment]
> @StevenSimon-NOAA , I can use the emc.vpppg stats for the plots. I just hadn't been since those stats do not have f000 calculated.
> 
> As for your personal archive, where there are holes, you can try to fill them w/ emc.vpppg files as a stop gap. Otherwise, like @ShannonShields-NOAA does, you need to stage the prep data in /dfs/write on cactus ahead of the switch. Then once the switch is complete, you can grab the prep data from /dfs/read on dogwood to populate your archive. One problem, though, is that the /dfs disk is limited. If you cannot get all of your prep data staged in /dfs/write on cactus (you can always regenerate stats), ask me and Alicia and we can try to stage the remainder in our /dfs/write spaces.

@ShelleyMelchior-NOAA Thanks for your recommendations. I am going to try staging my prep data on Cactus/dfs/write tomorrow. It looks like we will be having another dev/prod switch on Tuesday of the upcoming week. I will also fill in the prep file gaps in my local archive with emc.vpppg prep data where applicable. I suspect dev/prod switch associated gaps in my local prep archive may be the culprit for any gaps downstream with my locally generated stats and plots files derived from my local COMIN.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
This PR is approved for merge. Some outstanding problems will be addressed in subsequent forthcoming PRs. This PR needs to be merged presently so that f000 stats may begin to be generated in the EVS parallel.

**Pull request merged**